security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -0,0 +1,77 @@
++
+* Previously, only master node taints were tolerated. Now, the Compliance Operator tolerates all taints, making it possible to schedule scans on all nodes.
+
+
+[id="openshift-compliance-operator-0-1-44-bug-fixes"]
+=== Bug fixes
+
+* Previously, while performing a scan, a crash was occurring in one of the scanner containers of the pods performing the scan. In this release, the Compliance Operator uses the latest OpenSCAP version 1.3.5 to avoid this issue.
++
+* Previously, only one global `KubeletConfig` object was created per pool. In this release, a `KubeletConfig` object is created for each pool. 
++
+* The rules for tailored profiles are now validated so that only rules of the same type (Platform or Node) are included in the validation.
++
+* Remediations now have a `NeedsReview` state if they require a variable to be set. In order to automatically apply remediations, you must set all required variables before the MachineConfig pool is unpaused and the remediations applied.
++
+* Previously, the variable that was set for value-required handling was read from the datastream. Now, the variable is read directly from the tailoring to ensure that the appropriate remediation is created. 
++
+* The RBAC Role and RoleBinding used for Prometheus metrics is changed to `cluster-wide` to ensure that monitoring works out-of-the-box when using ClusterRole or ClusterRoleBinding for monitoring permissions.
++
+* Previously, if an error occurred while parsing a profile, rules or variables objects would be removed and deleted from the profile. Now, if an error occurs during parsing, the `profileparser` annotates the object with a temporary annotation that prevents the object from being deleted until after parsing completes. link:https://bugzilla.redhat.com/show_bug.cgi?id=1988259[(BZ#1988259)].
++
+* Because the XCCDF standard discourages empty titles and descriptions for tailored profiles, titles and descriptions are now required to be set in `TailoredProfile` CRs.
++ 
+* Previously, when using tailored profiles, `TailoredProfile` variable values were only allowed to be set using a specific selection set. This restriction is now removed, and `TailorProfile` variables can be set to any value.
+
+[id="compliance-operator-release-notes-0-1-39"]
+== Release Notes for Compliance Operator 0.1.39
+Issued: 2021-08-23
+
+The Compliance Operator 0.1.39 is now available with enhancements.
+
+[id="compliance-operator-0-1-39-new-features-and-enhancements"]
+=== New features and enhancements
+
+* Previously, the Compliance Operator was unable to parse Payment Card Industry Data Security Standard (PCI DSS) references. Now, the `profileparser` can parse compliance content that ships with PCI DSS profiles.
```

Comment:
Is it important to point out `profileparser`? Or, can we just say Operator?

Reply:
You're right, profileparser is just an implementation detail.